data/sensors_recent.php

```diff
@@ -357,11 +484,18 @@ function output_cell($rowspan, $data) {
 					}
 				}
 				$stationcount = count($stations);
-				echo("<tr><td>".gateway_button($gw_id)."</td><td>".htmlspecialchars($messagecount)."</td><td>".htmlspecialchars($stationcount)."</td><td>".$stationlist."</td></tr>\n");
+
+				$alias = "&nbsp;";
+				if (isset($ttn_gw_data[$gw_id]['alias']))
+					$alias = $ttn_gw_data[$gw_id]['alias'];
+				$url = "https://ttnmapper.org/gateways/?gateway=$gw_id&startdate=&enddate=&gateways=on&points=on";
```

Comment:
Goed idee om te linken naar ttnmapper, misschien goed om dat (ook) in `gateway_button` te doen?